src/plugins/intel_cpu/src/emitters/snippets/x64/jit_brgemm_copy_b_emitter.cpp

```diff
@@ -100,13 +98,10 @@ void jit_brgemm_copy_b_emitter::init_brgemm_copy(std::unique_ptr<matmul::jit_brg
     brgCopyKernelConf.tr_b_dt_sz = DnnlExtensionUtils::sizeOfDataType(static_cast<dnnl::memory::data_type>(brgCopyKernelConf.src_dt));
     brgCopyKernelConf.req_wei_vnni_downconvert = false;
 
-    if (is_with_amx) {
-        brgCopyKernelConf.isa = avx512_core_amx;
-        brgCopyKernelConf.s8s8_compensation_required = false;
-    } else {
-        brgCopyKernelConf.isa = src_dt == dnnl_data_type_t::dnnl_bf16 ? avx512_core_bf16 : avx512_core_vnni;
-        brgCopyKernelConf.s8s8_compensation_required = src_dt == dnnl_data_type_t::dnnl_s8;
-    }
+
+    brgCopyKernelConf.isa = jit_brgemm_emitter::get_primitive_isa(src_dt, is_with_amx);
+    OV_CPU_JIT_EMITTER_ASSERT(isa_has_int8_vnni(brgCopyKernelConf.isa), "BrgemmCopyB emitter requires VNNI support");
```

Comment:
Could you please clarify why we have to add this condition? I didn't find this limitation in onednn.
Moreover, if we want to support `transposed_b=true` via `brgemm_copy_b`, we will spend a lot of time to resolve conflicts between your and @v-Golubev PR because I think his changes don't need this check 😭